cfgmgr/teammgr.cpp

```diff
@@ -237,6 +238,16 @@ bool TeamMgr::checkPortIffUp(const string &port)
     return ifr.ifr_flags & IFF_UP;
 }
 
+bool TeamMgr::checkKernelPortMaster(const string &port)
```

Comment:
the name is so technical and it does not reflect the purpose. the purpose here is to find if the port is already enslave or not. so, better to rename the function as isPortEnslaved()